sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/immunizations/CreateNewImmunizationSteps.java

```diff
@@ -88,6 +88,17 @@ public CreateNewImmunizationSteps(
           selectResponsibleDistrict(immunization.getResponsibleDistrict());
           webDriverHelpers.clickOnWebElementBySelector(IMMUNIZATION_POPUP_SAVE_BUTTON);
         });
+
+    When(
+        "^I fill a new immunization form with specific responsible location$",
```

Comment:
here where the parametrized method will go, I would add region and district values into method naming, so the code will be readable and understand also the step(need) from scenario

Reply:
done